server/src/rest_api.rs

```diff
@@ -465,7 +465,7 @@ pub async fn register_vertiport(
         },
         location: payload.location,
         description: payload.description,
-        vertipads: payload.vertipads,
+        vertipads: Vec::new(),
```

Comment:
I think the vertipads key should be removed entirely here

Reply:
It's a required field for the vertiport struct. By definition, a vertiport must have at least one vertipad (itself). So here, it's an empty array. On the frontend, we won't let user proceed unless they insert at least one vertipad (they call also toggle something like "user this vertiport as a single vertipad").

Reply:
@GoodluckH @owlot I think there is potential use for a vertiport with no pads initially. A vertiport owner may want to appear on the network explorer some time in advance of the vertiports actual operation, during construction etc. where the pads haven't been inspected and certified yet

In the network explorer this would be an upcoming/coming soon vertiport that can advertise its existence in advance and drum up potential ridership, rather than popping into existence suddenly

There's otherwise no harm to having a vertiport with no pads, they just aren't included in routing since we route between pads

Reply:
[751867e](https://github.com/Arrow-air/svc-assets/pull/3/commits/751867ed5f9a2c4c7e57a741b97ff3187fa4465c)